bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_M-1000_V-0.01_tau_NLO/HeavyNeutrino_trilepton_M-1000_V-0.01_tau_NLO_madspin_card.dat

```diff
@@ -1,8 +1,8 @@
 set ms_dir ./madspingrid
 set Nevents_for_max_weigth 250
 set max_weight_ps_point 400
-set spinmode madspin
+set spinmode onshell
```

Comment:
spinmode onshell for mN=1000 GeV?

Reply:
Here you can just split the decays (into Z or W) with spinmode full.
Do you intend to use this?